sites/partners/pages/listings/[id]/edit.tsx

```diff
@@ -19,6 +19,15 @@ const NewListing = () => {
 
   if (!listingDto) return false
 
+  // Set listing photo from assets if necessary:
+  if (listingDto.image == null && listingDto.assets.length > 0) {
+    listingDto.image = listingDto.assets.find((asset) => asset.label == "building")
```

Comment:
In other places I see the label as `cloudinaryBuilding`, is it intentional that they don't match?

Reply:
yeah I switched the label for new Cloudinary-based uploads so we can distinguish between full file URLs (the old way) and Cloudinary IDs (the new way).

Reply:
> Why is this necessary and why do we want it to be dependent on the label === "building"?

Not sure! That was how it was before I dived into the code. Maybe it's deprecated and we can assume there's only ever a `listing.image` value?